ocp_resources/rhmi.py

```diff
@@ -8,3 +9,32 @@ class RHMI(NamespacedResource):
     """
 
     api_group = NamespacedResource.ApiGroup.INTEGREATLY_ORG
+
+    def wait_for_install_status_complete(self, timeout):
+        """
+        Wait until RHMI stage status is complete.
+
+        Args:
+            timeout (int): Time to wait for installation status.
+
+        Raises:
+            TimeoutExpiredError: If stage status is not complete.
+
+        """
+        self.logger.info(f"Wait for {self.kind} {self.name} installation status to be {RHMI.Status.COMPLETE}")
+        sample = None
+        try:
+            timeout_watcher = TimeoutWatch(timeout=timeout)
+            self.wait(timeout=timeout_watcher.remaining_time())
+
+            for sample in TimeoutSampler(
+                wait_timeout=timeout_watcher.remaining_time(),
+                sleep=3,
```

Comment:
why 3?

Reply:
This takes ~35 min to be in complete stage, so I added 3 here, we can change it to 1 also, good to me